frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "react-router-dom": "~4.3.1",
     "react-scripts": "~2.1.8",
     "reactstrap": "^7.1.0",
-    "universal-cookie": "~4.0.2"
+    "universal-cookie": "^4.0.2"
```

Comment:
I think we probably don't want to make changes to package.json/package-lock.json unless we actually changed some packages, so you can reset these changes.

Reply:
@vhoof we kept getting errors that it couldn't find the universal-cookie package. So when we installed it, is when it made changes to package.json. How do we reset this without getting an error again?

Reply:
Yeah, the package was added in staging. Since it's already in package.json, all you need to do is `docker-compose build` to update your dependencies.

Reply:
And you'll want to reset these files to their staging state first.